gradle.properties

```diff
@@ -13,3 +13,9 @@ kotlin.mpp.enableCInteropCommonization=true
 
 # BuildKonfig
 buildkonfig.flavor=staging
+
+#Native
+# Support multiple KMM modules for iOS project https://youtrack.jetbrains.com/issue/KT-42254
```

Comment:
You did not have multiple KMM modules currently, so don't need to have this setup

Reply:
I faced this error when removing these 🙏 
![CleanShot 2023-04-20 at 10 44 08](https://user-images.githubusercontent.com/17875522/233252910-7eb59134-3233-4d76-9b5e-5a4cd53cd783.png)

Reply:
@markgravity How about remove only `kotlin.native.cacheKind=none` ?

Reply:
@markgravity How about this?

Reply:
@markgravity Please remove the comment as well 😉